src/panels/lovelace/cards/hui-media-control-card.ts

```diff
@@ -35,6 +40,22 @@ import { fireEvent } from "../../../common/dom/fire_event";
 import { MediaControlCardConfig } from "./types";
 import { UNAVAILABLE } from "../../../data/entity";
 
+function luminanace(r: number, g: number, b: number): number {
+  const a = [r, g, b].map((v) => {
+    v /= 255;
+    return v <= 0.03928 ? v / 12.92 : Math.pow((v + 0.055) / 1.055, 2.4);
+  });
+  return a[0] * 0.2126 + a[1] * 0.7152 + a[2] * 0.0722;
+}
+
+function contrast(rgb1: number[], rgb2: number[]): number {
```

Comment:
same. `common/color/X.ts`

Reply:
Better to be explicit about the length of the array.

```suggestion
function contrast(rgb1: [number, number, number], rgb2: [number, number, number]): number {
```

Reply:
unresolving this comment because neither have been addressed

Reply:
I have resolved in my code. I check them as I go to clean up what I am looking at